web/frontend/src/pages/election/components/utils/useChangeAction.tsx

```diff
@@ -485,6 +501,24 @@ const useChangeAction = (
   };
 
   const getAction = () => {
+    // Check that more than 2/3 of the nodes are working, if not actions are
+    // disabled, since consensus cannot be achieved
+    const consensus = 2 / 3;
+
+    if (
+      Array.from(DKGStatuses.values()).filter((nodeStatus) => nodeStatus !== NodeStatus.Unreachable)
+        .length <=
+      consensus * roster.length
+    ) {
+      return <NoActionAvailable />;
+    }
+
+    // Except for seeing the results, all actions at least require the users
+    // to be logged in
+    if (!isLogged && status !== Status.ResultAvailable) {
+      return <NoActionAvailable />;
```

Comment:
Just a quick question: In this case is the user informed about why no action is available ? If not, would it be possible to add a message ?